components/ai_rewriter/resources/page/components/BeginGeneration.tsx

```diff
@@ -47,8 +47,7 @@ export default function BeginGeneration() {
             isMobile: false,
             hasAcceptedAgreement: true
           }}
-          onFocusInputMobile={() => undefined}
-          onBlurInputMobile={() => undefined} />
+          onFocusInputMobile={() => undefined} />
```

Comment:
Whilst we're editing this, please can we fix this issue that `onFocusInputMobile` isn't optional. Can be defined in `input.tsx` as
```
onFocusInputMobile?: () => unknown
```